F/FastJet_Julia_Wrapper/build_tarballs.jl

```diff
@@ -0,0 +1,53 @@
+# Note that this script can accept some limited command-line arguments, run
+# `julia build_tarballs.jl --help` to see a usage message.
+using BinaryBuilder
+
+name = "FastJetWrapBuilder"
+version_number = get(ENV, "TRAVIS_TAG", "")
+if version_number == ""
+    version_number = "v0.99"
+end
+version = VersionNumber(version_number)
+
+# Collection of sources required to build Fjwbuilder
+sources = [
+	   "FastJet_Julia_Wrapper"
+]
+
+# Bash recipe for building across all platforms
+script = raw"""
+wget https://github.com/JuliaPackaging/JuliaBuilder/releases/download/v1.0.0-2/julia-1.0.0-${target}.tar.gz
```

Comment:
Yeah, I don't disagree. Also applies to cxxwrap_julia...
Wouldn't this propagate the jll to the dependencies, though? How about being able to specify the jll as a source?
In this case, this would be the same as a "build-time dependency", but I don't know if that covers all cases.